test/Interop/Cxx/operators/Inputs/member-inline.h

```diff
@@ -6,4 +6,11 @@ struct IntBox {
   IntBox operator+(IntBox rhs) { return IntBox{.value = value + rhs.value}; }
 };
 
+struct HasUserProvidedCopyConstructor {
```

Comment:
Did you mean to provide a copy constructor?

Reply:
🤦 yep. And of course, now it crashes. Thanks for the quick review :P